modules/gdscript/gdscript.cpp

```diff
@@ -1779,10 +1779,11 @@ void GDScriptLanguage::get_reserved_words(List<String> *p_words) const {
 		"remote",
 		"sync",
 		"master",
+		"puppet",
 		"slave",
 		"remotesync",
 		"mastersync",
-		"slavesync",
+		"puppetsync",
```

Comment:
Wouldn't removing `slavesync` be a breaking change...? Or am I missing something about this change?

Reply:
`slavesync` was never backported to `3.0`, so it was never officially released.

Reply:
Ah, okay. Nevermind then.